packages/caliper-tests-integration/fabric_tests/phase3/src/marbles/node/marbles.js

```diff
@@ -40,8 +40,33 @@ let Chaincode = class {
      * @return {Promise<SuccessResponse>} Returns a promise of a response indicating the result of the invocation.
      */
     async Init(stub) {
-        let ret = stub.getFunctionAndParameters();
-        console.info(ret);
+        const marbles = [
+            {
+                name: 'marble1',
```

Comment:
missing `docType` field - check the `initMarble` method for details there (it's required for indexed searching on couchDB)